### PR TITLE
update endowed positions data

### PR DESCRIPTION
**Pull request by wmui51**
# READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- TL;DR - what's this PR for?
- Update endowed positions data

# Review By (Date)
- When does this need to be reviewed by?
- @forrestglick can confirm, but I believe this should be released on Tuesday 11/12

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Review Tasks

## Setup tasks and/or behavior to test

1. Check preview sites and append /endowed-positions/bass-university-fellows-in-undergraduate-education-program or any of the other schools and spot check, also make sure main site loads.

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

**Review by yvonnetangsu — APPROVED** [review]
Looks GTG to me. @forrest - ready for me to merge to `dev` for client review?

**Comment by forrestglick** [comment]
@wmui51 A few additional changes indicated in the original email from the client.

1. We need to add "University" into all references of Stanford University Libraries: https://deploy-preview-454--adapt-giving.netlify.app/endowed-positions/stanford-libraries/ – plse update copy on the page, along with the header, the url/slug (/stanford-university-libraries ), and the page title. 
2. The link in the copy should also be updated to point to: https://library.stanford.edu/support-stanford-libraries

Please make the updates, and I'll review before we push to Dev for client review. Thanks!
@yvonnetangsu is there a way we can create a redirect from the /stanford-libraries to the new url?

**Comment by yvonnetangsu** [comment]
> @wmui51 A few additional changes indicated in the original email from the client.
> 
> 1. We need to add "University" into all references of Stanford University Libraries: https://deploy-preview-454--adapt-giving.netlify.app/endowed-positions/stanford-libraries/ – plse update copy on the page, along with the header, the url/slug (/stanford-university-libraries ), and the page title.
> 2. The link in the copy should also be updated to point to: https://library.stanford.edu/support-stanford-libraries
> 
> Please make the updates, and I'll review before we push to Dev for client review. Thanks! @yvonnetangsu is there a way we can create a redirect from the /stanford-libraries to the new url?

Hi @forrestglick, I do see a redirect directory/component on the Giving site so that should't be a problem. I'll wait till the  updates are ready 👍🏼

**Comment by forrestglick** [comment]
Updates look good @wmui51
@yvonnetangsu – please push to Dev for client review. Thanks!

**Comment by yvonnetangsu** [comment]
Merge to dev @forrestglick! Please give it a few minutes :)

**Comment by forrestglick** [comment]
The client has reviewed and approved the update. She would like it pushed tomorrow (Tuesday) morning, if possible. Thank you!

**Comment by forrestglick** [comment]
I also successfully tested the redirect on Dev and will set that up on Prod tomorrow.

**Comment by yvonnetangsu** [comment]
Sure, we can prep a release to be pushed live tomorrow morning 👍🏼 @forrestglick
